common/src/main/java/gov/cms/ab2d/common/util/FilterOutByDate.java

```diff
@@ -288,6 +289,14 @@ static boolean withinDateRange(ExplanationOfBenefit ben, DateRange range) {
         if (start == null || end == null) {
             return false;
         }
+
+
+        // May 1st - June 1st
```

Comment:
This is a member in April?

This whole comment block seems disjointed, perhaps it is just a formatting issue but it makes it harder for me to understand what is going on here.  That's the last thing that a comment should do.